src/coreclr/nativeaot/Runtime/windows/PalRedhawkMinWin.cpp

```diff
@@ -671,6 +671,42 @@ REDHAWK_PALEXPORT char* PalCopyTCharAsChar(const TCHAR* toCopy)
     return converted;
 }
 
+REDHAWK_PALEXPORT HANDLE PalLoadLibrary(const char* moduleName)
+{
+    if (moduleName == nullptr)
```

Comment:
Why do we need to treat null?

Reply:
So far, no call site will pass `null`, so it is just defensive programming. I can easily remove it.